cmd/insert-user/main.go

```diff
@@ -53,21 +68,25 @@ func main() {
 
 	flag.Parse()
 
-	if len(os.Args) != 3 {
-		fmt.Fprintf(os.Stderr, "usage: %s <user-name> <@theirPublicKey.ed25519>\n", os.Args[0])
-		flag.Usage()
-		os.Exit(1)
-		return
+	// we require at least 5 arguments: <executable> + -name <val> + -key <val>
+	//                               1              2     3       4    5
+	if len(os.Args) < 5 {
```

Comment:
I think this is fine but the comment tells me something is foul here.

There is https://pkg.go.dev/flag#Args which can be used to get the "non flag args", after flag.Parse() of course.

Obviously if we use it it would be zero and we still need a way to check that `-name` and `-key` were passed and counting the args doesn't tell us which those were either soo.. again. Might be fine.

Just putting it out there for future uses of the flag package.